.codecov.yml

```diff
@@ -0,0 +1,23 @@
+coverage:
+  status:
+    project:
+      default: off
+      author:
+        flags: author
+      api:
+        flags: api
+      publisher:
+        flags: publisher
+
+flags:
+  # filter the folder(s) you wish to measure by that flag
```

Comment:
I think these comments are unnecessary

Reply:
Yeah, this is just a copy and paste from the docs at the moment to see what we can get it to do at the moment.